src/ngx_lua_resty_lmdb_module.c

```diff
@@ -31,6 +48,20 @@ static ngx_command_t  ngx_lua_resty_lmdb_commands[] = {
       offsetof(ngx_lua_resty_lmdb_conf_t, map_size),
       NULL },
 
+    { ngx_string("lmdb_encryption_key_data"),
```

Comment:
We should not keep secret in nginx conf, maybe we could change it to a file in next PR.

Reply:
that's a good point, however let's move this to the post feature freeze, to give us more time to implement this.

Reply:
done #19